launch/MRS1000.launch

```diff
@@ -1,8 +1,17 @@
 <launch>
   <arg name="host" default="192.168.2.140" />
   <arg name="port" default="2111" />
+  <arg name="range" default="64.0" />
+  <!-- first, last, all -->
```

Comment:
no "strongest" option here?

Reply:
This is for configuring the sensor, those are the only options the hardware provides. "Strongest" is done in software by the ros node by sorting "all" echoes by reflectance.